### PR TITLE
add example run scripts

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: doc/running.rst [hunk 1]
```diff
@@ -88,6 +91,14 @@ An example batch script for the slurm batch system is given below:
...
+   # Run this script with 'sbatch <script.sh>'
+
+   # Put any environment activation here, e.g.:
+   # source ../../config/activate_env.sh
+
+   # Put any device selection here, e.g.:
+   # export PYOPENCL_CTX=":"
+
    nnodes=$SLURM_JOB_NUM_NODES
    nproc=$nnodes # 1 rank per node
```
Comment (inducer) [inline doc/running.rst:104]: This bit looks pretty redundant with the script in examples.

How about
```rst
.. literalinclude:: ../examples/lassen.bsub.sh
    :language: bash
```
instead? (Also below)

File: doc/running.rst [hunk 2]
```diff
@@ -65,6 +65,9 @@ Note that profiling has a performance impact (~20% at the time of this writing).
 Running on specific systems
 ---------------------------
 
+This section discusses how to run mirgecom on various clusters.
+There are also several example run scripts in mirgecom's ``examples/`` folder.
```
Comment (inducer) [inline doc/running.rst:69]: Link to it on Github?

File: doc/running.rst (hunk 1 above)
Comment (matthiasdiener) [reply doc/running.rst]: good idea, changed.

File: doc/running.rst (hunk 2 above)
Comment (matthiasdiener) [reply doc/running.rst]: done

**Comment by majosm** [comment]
Do we have anything in the documentation that describes how to set `PYOPENCL_CTX` to select the GPUs on Lassen? If not, I think that would be good to add (or maybe we could just set it that way by default in the example script?).

**Comment by matthiasdiener** [comment]
> Do we have anything in the documentation that describes how to set `PYOPENCL_CTX` to select the GPUs on Lassen? If not, I think that would be good to add (or maybe we could just set it that way by default in the example script?).

I don't think we have that documentation, but it would be nice to add. Imho we shouldn't really set anything here, since we don't know what the user wants or has installed.

File: doc/running.rst [hunk 3]
```diff
@@ -65,6 +65,10 @@ Note that profiling has a performance impact (~20% at the time of this writing).
 Running on specific systems
 ---------------------------
 
+This section discusses how to run mirgecom on various clusters.
+There are also several example run scripts in mirgecom's ``examples/``
+`folder <https://github.com/illinois-ceesd/mirgecom/tree/master/examples>`_.
```
Comment (inducer) [inline doc/running.rst:70]:
```suggestion
`folder <https://github.com/illinois-ceesd/mirgecom/tree/master/examples>`__.
```

**Review by inducer — APPROVED** [review]

**Pull request merged**